intake/source/base.py

```diff
@@ -62,103 +187,75 @@ class DataSource(DictSerialiseMixin):
 
     This class is not useful in itself, most methods raise NotImplemented.
     """
+
     name = None
     version = None
     container = None
     partition_access = False
-    datashape = None
```

Comment:
Why did `datashape` get removed?

Reply:
It's not used anywhere at all, is always None